src/PDFsharper.UnitTests/Pdf.IO/ParserTests.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PdfSharper.Pdf;
+using PdfSharper.Pdf.IO;
+using PdfSharper.Pdf.Filters;
+using System.Linq;
+using System.IO.Compression;
+using System.IO;
+using PdfSharper.Pdf.AcroForms;
+using PdfSharper.Pdf.Internal;
+using System.Diagnostics;
+
+namespace PDFsharper.UnitTests.Pdf.IO
+{
+    [TestClass]
+    public class ParserTests
+    {
+
+#if DEBUG && PEX
```

Comment:
These are only for running here - i can remove them from the solution entirely if deemed appropriate.